plugins/tuna-obs/back.js

```diff
@@ -41,7 +41,7 @@ module.exports = async (win) => {
 
 		data.duration = secToMilisec(songInfo.songDuration)
 		data.progress = secToMilisec(songInfo.elapsedSeconds)
-		data.cover_url = songInfo.imageSrc;
+		data.cover = songInfo.imageSrc;
```

Comment:
Could we set both cover and cover_url? This way, it would support all versions (I guess additional properties are not an issue?)